javascript/javascript1/week4/infoCreditCard.js

```diff
@@ -0,0 +1,26 @@
+//Credit card info
+
+let number = '4781321334789876';
```

Comment:
If i set the number to `4026321334789876` which is a visa electron card. what is returned form your function?